docs/user-guide/security.md

```diff
@@ -116,27 +116,28 @@ There is one advantage of Directed userns ID mapping:
     This means Sysbox can run in kernels that don't carry that module (e.g.,
     Ubuntu cloud images).
 
-But there is a drawback:
+But there are a couple of drawbacks:
 
 -   Configuring Docker with userns-remap places a few [functional limitations](https://docs.docker.com/engine/security/userns-remap/#user-namespace-known-limitations)
     on regular Docker containers (those launched with Docker's default runc).
 
+-   Docker assigns the same user-ID mapping to all containers. This is not ideal:
+    it would be better to assign each container exclusive user-ID mappings for
+    improved cross-container isolation.
```

Comment:
This point is only relevant in sysbox-EE and not in sysbox-CE so this sentence may need clarification.

Reply:
Fixed, thanks!